frontend/src/pages/accounts/settings.module.scss

```diff
@@ -141,6 +158,7 @@ $field-gap: $spacing-lg;
 
   .copy-button {
     display: block;
+    padding-left: $spacing-md;
```

Comment:
Could you instead add this as a `gap` on `.settings-api-key-wrapper`? To have the parent determine the layout of its children, rather than children assuming the elements they're being used in.